maintenance/MaintenanceReferenceOntology.rdf

```diff
@@ -465,6 +484,7 @@
 	
 	<owl:ObjectProperty rdf:about="&iof-maint;stateOf">
 		<rdfs:subPropertyOf rdf:resource="&obo;BFO_0000167"/>
+		<rdfs:label>maintainableMaterialStateOf</rdfs:label>
```

Comment:
Labels should be readable by humans, so in this case "maintainable material state of" might be better.